<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Status</class>
 <widget class="QWidget" name="Status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="PlotCustom" name="ratePlot" native="true">
     <zorder>statusValuesGroupBox</zorder>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="statusValuesGroupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <widget class="QLabel" name="ADCLabel1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ADC Ch1: </string>
      </property>
     </widget>
     <widget class="QLabel" name="ADCLabel2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ADC Ch2: </string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotCustom</class>
   <extends>QWidget</extends>
   <header>plotcustom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
